src/data.py

```diff
@@ -0,0 +1,428 @@
+
+    Kulcs (int): Toplista helyezés sorrendje, számlálóval kerül megadásra.
+    Érték (str): Part Number, Brand, Model, Rank, Benchmark, Samples, URL
+    A Rank egyforma benchmark érték estén, több alkatrésznél is lehet ugyan az.
+
+    >>> print_data({1:["RTX 4090 SUPRIM Liquid X 24G","MSI","MSI RTX 4090 24GB Suprim Liquid X",1,370,25956,"https://gpu.userbenchmark.com/Nvidia-RTX-4090/Rating/4136"]}, 1)
+    MSI RTX 4090 24GB Suprim Liquid X. RTX 4090 SUPRIM Liquid X 24G MSI, Score: 1, Tesztek száma:370
+    >>> print_data({1: ["F5-6400J3239G32GX2-TR5S","G.SKILL","TZ/RJ/FX DDR5 6400 C32 2x32GB",4,188,6571,"https://ram.userbenchmark.com/SpeedTest/1963529/G-Skill-Intl-F5-6400J3239G32G-2x315GB"], 2: ["F5-6400J3239G16GA2-TZ5RS","G.SKILL","TZ/RJ/FX DDR5 6400 C32 2x16GB",5,187,18403,"https://ram.userbenchmark.com/SpeedTest/1770998/G-Skill-Intl-F5-6400J3239G16G-2x16GB]"], 3: ["F5-6400J3239G16GA2-TZ5S","G.SKILL","TZ/RJ/FX DDR5 6400 C32 2x16GB",5,187,18403,"https://ram.userbenchmark.com/SpeedTest/1770998/G-Skill-Intl-F5-6400J3239G16G-2x16GB"]}, 3)
+    TZ/RJ/FX DDR5 6400 C32 2x32GB. F5-6400J3239G32GX2-TR5S G.SKILL, Score: 4, Tesztek száma:188
+    TZ/RJ/FX DDR5 6400 C32 2x16GB. F5-6400J3239G16GA2-TZ5RS G.SKILL, Score: 5, Tesztek száma:187
+    TZ/RJ/FX DDR5 6400 C32 2x16GB. F5-6400J3239G16GA2-TZ5S G.SKILL, Score: 5, Tesztek száma:187
+    """
+    COUNTER = 0
+    for value in data.values():
+        if COUNTER == int(lines):
+            return
+        else:
+            COUNTER += 1
+            TEMP = str(value[1]).split(" ")
+            if (value[0]) == TEMP[0]:
+                print (f'{value[2]}. {value[1]}, Score: {value[3]}, Tesztek száma:{value[4]}')
+            else:
+                print (f'{value[2]}. {value[0]} {value[1]}, Score: {value[3]}, Tesztek száma:{value[4]}')
+    return
+
+
+def custom_partdata(filter: list[str], number: int, menu: str) -> None:
+    """Kiválasztott konkrét alkatrész adatainak kiíratása.
+
+    Args:
+        filter: list[str]: Szűrésre megadott feltétel, None esetén nem történik szűrés.
+        number (int): PARTS konstans hanyadik eleme alapján történjen az adat betöltés.
+        menu: (str): Melyik menüpontról került meghívásra.
+
+    2-es menü esetén kiíratja a konkrét alkatrész adatai által az elérhető értékeket.
+    Igen érték vagy 3-as menü esetén átadja az adatokat a diagramm elkészítéshez.
+    """
+    with open(f"data/benchmark_description.json", encoding="utf-8") as f:
+        bench_desc = json.load(f)
+    for part in bench_desc:
+        for key, value in part.items():
+            if key == PARTS[number - 2]:
+                filtered_desc = value    
+    part_data = load_data(PARTS[number - 2])
+    filtered_data = filter_data(number, filter)
+    for key, values in filtered_data.items():
+        print (f"{key}. - {values[0]} {values[1]}")
+    choice = int(input("Válassz a találatok sorszámai közül: "))
+    keys = [int(key) for key in part_data.keys()]
```

Comment:
A `part_data` kulcsai eleve intek.
```suggestion
    keys = list(part_data.keys())
```

Reply:
Korrigálva.